website/.spelling

```diff
@@ -552,7 +552,8 @@ defaultMetrics.json
 namespacePrefix
 src
  - ../docs/development/extensions-contrib/redis-cache.md
-loadList
```

Comment:
`loadList` is not referenced in the doc, so it's useless

Reply:
Indeed, there is no loadList in this document, but there is still in other documents. This .spelling should be global, not specific to a particular document.

Reply:
Indeed, there is no `loadList` in this document, but it still exists in other documents. We can adjust the position of `loadList` in the `.spelling` file, but it is recommended to keep it.

Reply:
Is the `loadList` here a global configuration for all files ? I thought it was related to `redis-cache.md` doc only because it was below that file path. 

BTW, I checked all files, no `loadList` is used but `druid.extensions.loadList`

Reply:
The `loadList` should be placed elsewhere in the `.spelling` file, not under the `redis-cache.md` document. Also, `loadList` is used by other document as a single word. I guess you may have missed the document `docs/configuration/index.md`. So I recommend adjusting the position of `loadList` in the `.spelling` file instead of deleting it.

Reply:
Ah I see. I misunderstood the rules in this `.spelling` file